src/main/java/io/jenkins/plugins/pipelinegraphview/PipelineGraphDisplayURLProvider.java

```diff
@@ -2,12 +2,13 @@
 
 import edu.umd.cs.findbugs.annotations.NonNull;
 import hudson.Extension;
-import hudson.Util;
+import hudson.model.Job;
 import hudson.model.Run;
-import org.jenkinsci.plugins.displayurlapi.ClassicDisplayURLProvider;
+import org.jenkinsci.plugins.displayurlapi.DisplayURLProvider;
+import org.jenkinsci.plugins.workflow.job.WorkflowRun;
 
-@Extension
-public class PipelineGraphDisplayURLProvider extends ClassicDisplayURLProvider {
+@Extension(ordinal = 50) // Take precedence over Blue Ocean if both are installed and there is no configured provider.
```

Comment:
If you want to not take precedence over Blue Ocean, we can make this less than 0.